openspec/changes/archive/2026-07-02-rfe-8685/inputs/scope-constraints.md

```
@@ -0,0 +1,13 @@
+# Scope constraints
+
+## Jira
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)